wimm.Secundatives.UnitTests/Result_Test.cs

```diff
@@ -92,9 +90,7 @@ public void Ok_ContainsError_ReturnsNone()
             var underTest = ConstructErr();
             Assert.False(underTest.Ok().Exists);
         }
-
-
-        private Result<int> ConstructInt()
+        private static Result<int> ConstructInt()
```

Comment:
Did you mean to leave no empty lines here?

Reply:
I did not